src/runtime/Makefile

```diff
@@ -541,9 +576,14 @@ USER_VARS += DEFBINDMOUNTS
 USER_VARS += DEFVFIOMODE
 USER_VARS += BUILDFLAGS
 USER_VARS += DEFSERVICEOFFLOAD
+USER_VARS += DEFCONFIDENTIALGUEST
```

Comment:
Drop this one, please, based on the previous comment,

Reply:
done